.github/workflows/pytest.yaml

```diff
@@ -0,0 +1,38 @@
+name: python-tests
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+  push:
+    branches:
+      - main
+      - feature/*
+
+concurrency:
+  group: '${{ github.workflow }}-${{ github.event_name }}-${{ inputs.platform }} @ ${{ github.event.pull_request.number || github.sha }}'
+  cancel-in-progress: true
+
+permissions: {}
+
+jobs:
+  python-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.7'
+      - name: Install Pip Packages
+        run: pip install -r ${GITHUB_WORKSPACE}/docker/pytest/requirements.txt
+      - name: Run Tests
+        run: coverage run --omit=*_test.py -m pytest
```

Comment:
Why do we need the --omit flag?

Reply:
Otherwise the test files are included in the coverage report. This is the case currently on buildbot.